extension/content.js

```diff
@@ -4,7 +4,7 @@ if (!alreadyInjected) {
 	
 	var alreadyInjected = true;
 
-	var WANTED_WIDTH = 800;
+	const WANTED_WIDTH = 800;

```

Comment:
Ich meine mich zu erinnern, dass das irgendwelche Probleme gab. Hast Du das getestet?

Reply:
So I tried running the extension and it seemed to work, but let me check it again doing the whole process.

Reply:
Unfortunately, we don't have automated tests...

Reply:
According to https://www.w3schools.com/js/js_const.asp, this should be no problem with the major browsers. But you could always use babel to transform this code to a lower versions. Maybe still a problem for IE, but this will only run in chrome right now.

Reply:
As for variable declarations: `const` -> `let` -> `var` (see [here](https://dev.to/sarah_chima/var-let-and-const--whats-the-difference-69e))

Reply:
I can generate a GM and a Report (diffing the same page against the GM) with and without the changes and it seems to work. At the first run I had some changes, but when I retried they were not present (the page I tested does use some CSS animation, maybe I tripped it?)

Do you remember what exactly kind of problems where caused by this change?

Reply:
Unfortunately not. But I probably mix that up with recheck-web and IE11...